### ISSUE TITLE
dtcw.bat fails to retrieve the date on a German Windows 10 installation

### ISSUE THREAD

**Issue by Timmeey86** [docToolchain/docToolchain#1267]
**Describe the bug**
The line 

```PS
$container_name="doctoolchain-${version}-$(date '+%Y%m%d_%H%M%S')"
```

 in the dtcw.bat fails on a German Windows with a message stating (free translation) 

> Get-Date : The parameter "Date" can not be bound. The value "+%Y%m%d_%H%M%S" can not be converted to type "System.DateTime". Error: "The input string was not recognized as a valid DateTime".

Changing the line to

```PS
$container_name="doctoolchain-${version}-$(date -Format 'yyyyMMdd_HHmmss')"
```

fixed it for me.
Note that I haven't tested if the format string leads to ambiguous container names.

**To Reproduce**
1. Download dtcw.bat from the official link onto a Windows System with German localization settings
2. Execute `dtcw tasks --group=doctoolchain` 
3. A CannotConvertArgumentNoMessage exception will occur

**Configuration**
 - docToolchain version 3.0.2
 - OS: Windows 10 Version 21H2 Build 19044.2965 (in an offline environment)
 - PSVersion 5.1.19041.2673

**Comment by timo-abele** [comment]
Maybe this is already fixed by #1261. Could you please see if the code change introduced [there](https://github.com/docToolchain/docToolchain/pull/1261/files):
````PS
$container_name="doctoolchain-${version}-$(date -uFormat '+%Y%m%d_%H%M%S')"
````
works for you?

**Comment by Timmeey86** [comment]
I can confirm that the introduced change works. I'll close the issue as I suppose the provided .bat file will be updated shortly anyway
